Cargo.toml

```diff
@@ -13,13 +13,16 @@ keywords = ["fft"]
 [dependencies]
 num-complex = "0.4"
 dyn-stack = { version = "0.8", default-features = false }
+pulp = "0.10"
+bytemuck = "1.13"
 aligned-vec = { version = "0.5", default-features = false }
 serde = { version = "1.0", optional = true, default-features = false }
 
 [features]
-default = ["std"]
-nightly = []
+default = ["std", "fft128"]
```

Comment:
> I did not have time to read everything, do you have some paper resource on the f128 format/approach you take ?

there's a paper and a reference implementation in c++

https://web.mit.edu/tabbott/Public/quaddouble-debian/qd-2.3.4-old/docs/qd.pdf

https://github.com/scibuilder/QD/

Reply:
i don't know if we want to enable it by default or not. i don't think it makes much of a difference since it's only used in tfhe-rs according to crates.io

Reply:
doesn't it mean the final binary has the fft128 code even if it goes unused ?

Reply:
i don't think so. i think the linker discards unused functions but i'm not sure

Reply:
So after thinking about this again I think it's better to have only the std feature enabled by default (not exposing fft128) and leaving the choice up to us (to limit the dependency tree as we have seen that it may cause some issues for integration)